src/Microsoft.DotNet.Interactive.CSharp/AspNetKernelExtensions.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.CSharp
```

Comment:
I suspect that this code will be more at home in the `Microsoft.DotNet.Interactive.Http` project.